dist/object-describer.js

```diff
@@ -141,7 +141,12 @@ angular.module('kubernetesUI')
     scope: {
       containers: '='
     },
-    templateUrl: 'views/containers.html'
+    templateUrl: 'views/containers.html',
+    link: function($scope, element, attrs) {
+      $scope.shouldMask = function(name) {
```

Comment:
Hmmm, the masking is just about masking something with bullets (ie: a password), not about the length. So if I'm understanding your comment right, then no, it shouldn't.